dvc/utils/pkg.py

```diff
@@ -46,7 +46,7 @@ def get_package_manager():
     import platform
     from dvc.exceptions import DvcException
 
-    if is_conda():
+    if check_build_patch() == "conda":
```

Comment:
How about we make this simpler. Just make `get_package_manager` return the `PKG` imported from `pkg.py`.

Reply:
That way we could get rid of other code :slightly_smiling_face: